xml/System.CodeDom.Compiler/IndentedTextWriter.xml

```diff
@@ -2398,11 +2398,11 @@
         <Parameter Name="count" Type="System.Int32" Index="2" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <param name="index">To be added.</param>
-        <param name="count">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">The buffer to write from.</param>
```

Comment:
```suggestion
        <param name="buffer">The character array to write data from.</param>
```

Reply:
I decided to leave some of these -- I don't think the purpose of the intellisense text is to say what the type is -- the tool (VS etc) shows that. It's to say what it's used for, or what it contains. As it's a text writer, any buffer to write from is going to be a string/char/span<char>/stringbuilder.

Reply:
Sounds good